modules/xmpp/TraceablePeerConnection.js

```diff
@@ -12,9 +12,26 @@ var SDPUtil = require("./SDPUtil");
 
 var SIMULCAST_LAYERS = 3;
 
-function TraceablePeerConnection(ice_config, constraints, session) {
+/**
+ * Creates new instance of 'TraceablePeerConnection'.
+ *
+ * @param {object} ice_config WebRTC 'PeerConnection' ICE config
+ * @param {object} constraints WebRTC 'PeerConnection' constraints
+ * @param {object} options <tt>TracablePeerConnection</tt> config options.
+ * @param {boolean} options.disableSimulcast if set to 'true' will disable
+ * the simulcast
+ * @param {boolean} options.disableRtx if set to 'true' will disable the RTX
+ * @param {boolean} options.preferH264 if set to 'true' H264 will be preferred
+ * over other video codecs.
+ * @param {EventEmitter} eventEmitter the emitter which wil be used by the new
+ * instance to emit events.
+ *
+ * @constructor
+ */
+function TraceablePeerConnection(ice_config,
+                                 constraints, options, eventEmitter) {
```

Comment:
wow, so we can remove all reference to JingleSession from TPC completely?  that's awesome...huge decoupling right there.